fenix/app/src/main/res/layout/activity_home.xml

```diff
@@ -1,26 +1,26 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<org.mozilla.fenix.perf.HomeActivityRootLinearLayout
+<FrameLayout
```

Comment:
I needed to make the root View the FrameLayout instead, in order to get the drawer UI over the top of the toolbar in the screens that use it (Settings, history, etc.)

Reply:
This makes me a little wary still, though thinking about it I don't think it should have any real impact on the perf metrics that `HomeActivityRootLinearLayout` tracks. 

I think we should include some strongly worded comments about not adding anything that isn't strictly debug drawer related to this view, though.

Reply:
For sure. The other upside is it removes the ComposeView from `HomeActivityRootLinearLayout`, so the overlay can't be accidentally included in any of the performance calculations now.

I'll add a comment to this file explaining what's up and a note on how to maybe handle this if this whole file is moved to Compose

Reply:
Little late to this, so for my understanding we're asserting that we are currently excluding the debug drawer from the performance profiling?

Reply:
Little late to this, so for my understanding we're asserting that we are currently excluding the debug drawer from the performance profiling?
Do we have a good understanding of the ramifications of making this change?

Reply:
Adding a note now.

It's only being excluded from any of the automatic profiling that currently exists on `HomeActivity`. We can still do anything manual or set up new profile markers just for the debug drawer in the future.